src/config/dynamic-theme-fixes.config

```diff
@@ -2012,6 +2012,18 @@ INVERT
 
 ================================
 
+matsci.org
+
+INVERT
+.d-header #site-logo
+
+CSS
+.category-logo.aspect-image img {
+  background-color: white !important;
```

Comment:
Use ${black} and check if that looks still good

Reply:
Unfortunately not